test/extended/networking/egress_firewall.go

```diff
@@ -128,6 +192,17 @@ func sendEgressFwTraffic(f *e2e.Framework, oc *exutil.CLI, pod string, nodeSelec
 	_, err = oc.Run("exec").Args(pod, "--", "curl", "-q", "-s", "-I", "-m3", "https://docs.openshift.com").Output()
 	expectNoError(err)
 
+	// TODO:
+	// - Check for coredns-ocp-dnsnameresolver plugin.
+	// - Remove TecHPreview check when feature is GA.
+	if networkPluginName() == OVNKubernetesPluginName && exutil.IsTechPreviewNoUpgrade(oc) {
+		// Test curl to registry.build05.ci.openshift.org should pass
+		// because we have allow dns rule for *.ci.openshift.org
+		g.By("sending traffic that matches allow dns rule")
+		_, err = oc.Run("exec").Args(pod, "--", "curl", "-q", "-s", "-I", "-m3", " https://registry.build05.ci.openshift.org").Output()
+		expectNoError(err)
+	}
+
```

Comment:
We can utilize the existing curl to `www.google.com` rather than adding a new curl here. Also, the current support added for wildcard DNS name is to check for only one level of subdomain. Thus, `*.ci.openshift.org` will not match `registry.build05.ci.openshift.org` as it has 2 levels of subdomain after `ci.openshift.org`.

Reply:
I have missed it. I had understood wildcards could be used only at the beginning and there was no restriction on the depth.